src/sentry/web/urls.py

```diff
@@ -146,6 +148,16 @@
         ErrorPageEmbedView.as_view(),
         name="sentry-error-page-embed",
     ),
+    re_path(
+        r"^organizations/(?P<organization_slug>[^/]+)/toolbar/iframe$",
+        IframeView.as_view(),
+        name="sentry-toolbar-iframe",
+    ),
+    re_path(
+        r"^organizations/(?P<organization_slug>[^/]+)/toolbar/login-success/$",
+        LoginSuccessView.as_view(),
+        name="sentry-toolbar-login-success",
+    ),
```

Comment:
These could go down near line 888 where it defines the `organizations` prefix:
https://github.com/getsentry/sentry/blob/6e85c069edd4881d52171a39ead697bea538fb1d/src/sentry/web/urls.py#L887-L889

And the `name` prop could be updated to match the format that's in there, `sentry-organization-foo-bar`